packages/core/admin/admin/src/pages/Settings/pages/Webhooks/EditPage.tsx

```diff
@@ -116,7 +116,7 @@ const EditPage = () => {
           message: formatMessage({ id: 'Settings.webhooks.created' }),
         });
 
-        navigate(res.data.id, { replace: true });
+        navigate(`/settings/webhooks/${res.data.id}`, { replace: true });
```

Comment:
tiny nitpick but I think a relative link (e.g. `../${res.data.id}`) gives us more flexibility if the parent routes ever change